front/src/Turtle.js

```diff
@@ -0,0 +1,109 @@
+import PropTypes from "prop-types";
+import React from "react";
+import Dropdown from "./Dropdown";
+import Battery from "./Battery";
+import styled from "styled-components";
+
+const TEAM_VALUES = ["Magenta", "Cyan"];
+const HOME_VALUES = ["Yellow home", "Blue home"];
+const ROLE_VALUES = [
+  "INACTIVE",
```

Comment:
Naming is not consistent, but this is due to the Tech United naming (it was copied)